Library/Homebrew/formulary.rb

```diff
@@ -738,6 +738,15 @@ def self.try_new(ref, from: nil, warn: false)
       return unless path.expand_path.exist?
       return unless ::Utils::Path.loadable_package_path?(path, :formula)
 
+      # If the path is for an installed keg, use FromKegLoader instead
```

Comment:
Totally fine to merge as-is for now but feels like ideally `FromKegLoader` would be higher in the load order and be detected as an option before `FromPathLoader`?